core/src/main/java/org/jdbi/v3/core/statement/SqlStatements.java

```diff
@@ -60,6 +60,9 @@ public final class SqlStatements implements JdbiConfig<SqlStatements> {
 
     private final Collection<StatementContextListener> contextListeners;
 
+    private int jfrSqlMaxLength = 512;
```

Comment:
is 512 a particular value or just "this looks ok". Adding a comment might be useful.

Reply:
Picked a reasonable number out of a hat. The intent is to not accidentally serialize megabytes of parameter data out via telemetry.